src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -106,24 +103,19 @@ export class ResultsViewPageStore {
         }
     }, []);
 
-    readonly mskImpactGermlineConsentedPatientIds = remoteData({
-        await: () => [this.patientIds],
+    readonly germlineConsentedSampleIds = remoteData({
         invoke: async () => {
-            const clinicalDataSingleStudyFilter = {
-                attributeIds: [IMPACT_GERMLINE_TESTING_CONSENT],
-                ids: this.patientIds.result
-            };
-            const clinicalDataResponse = await fetchClinicalDataInStudy(
-                this.studyId, clinicalDataSingleStudyFilter, 'PATIENT'
-            );
-            if (clinicalDataResponse) {
-                return _.uniq(clinicalDataResponse.map(
-                    (cd:ClinicalData) => cd.entityId)
-                );
+            if (this.germlineSampleListId) {
+                return await client.getAllSampleIdsInSampleListUsingGET({
+                    sampleListId: this.germlineSampleListId
+                });
             } else {
                 return [];
             }
         },
+        onError: () => {
+            // fail silently
```

Comment:
I don't think we need to fall back to `12_245_PARTC_CONSENTED`. Patients with the `12_245_PARTC_CONSENTED` info should also be in the `${this.studyId}_germline` sample list.

Here actually, I wanted to point out that for most of the studies (for now studies other than MSK-IMPACT) don't have  `${this.studyId}_germline` (neither they have `12_245_PARTC_CONSENTED`, I think). But since this solution is generic, it queries all studies no matter what, and we got a `not found` error from API and then conclude that there is no such sample list (which should be the correct conclusion). What I am saying is that we depend on an API error to reach that conclusion.

Maybe we specifically check the error type and make sure it is a `not found` error, but not something else?